src/main/java/com/mateacademy/springmvc/controller/UserController.java

```diff
@@ -0,0 +1,58 @@
+package com.mateacademy.springmvc.controller;
+
+import com.mateacademy.springmvc.entity.User;
+import com.mateacademy.springmvc.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
```

Comment:
It is really should be rest controller?

Reply:
actually yes i agree woth u) @controller is enough here